lib/sqlalchemy/ext/asyncio/engine.py

```diff
@@ -82,6 +82,21 @@ def create_async_engine(url: Union[str, URL], **kw: Any) -> AsyncEngine:
             "streaming result set"
         )
     kw["_is_async"] = True
+    async_creator = kw.pop("async_creator", None)
+    if async_creator:
+
```

Comment:
let's place a `if kw.get("creator", None): raise ArgumentError` mentioning that you cannot pass both async_creator and creator.


Also async_creator should be documented in this function

Reply:
Good suggestion, will add this